src/ingestion/process-event.ts

```diff
@@ -75,6 +75,12 @@ export class EventsProcessor {
         const personUuid = new UUIDT().toString()
 
         const ts = this.handleTimestamp(data, now, sentAt)
+        // Send latency estimate to statsd, but ignore events set in the future.
+        const latencyEstimate = (sentAt || ts).toMillis() - singleSaveTimer.getMilliseconds()
```

Comment:
AFAIK (I may be wrong) sent_at is provided by client libraries, so I'm not sure if this is v. useful?

Reply:
I agree, this could get pretty noisy if a user is backfilling historical data. It would make us look way behind when in fact it's just history being submitted after the fact.

Might be worth setting a TS when we start processing a batch or events as they come in and check as we are producing what the diff is between now and the batch start TS